### PR TITLE
IMPROVEMENT: Add missing codeblock.

### PR DESCRIPTION
**Pull request by crydotsnake**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,14 +4,20 @@ This package allows you to display information about your environment (or any ot
...
 ![Demo View](Documentation/Demo.png)
+
 ## Install
-Install via composer (composer2 is required):
-`composer require punktde/neos-environment-display`
+
+Install via composer2:
+
+```shell
+$ composer require punktde/neos-environment-display
```
Comment (simonschaufi) [inline README.md:13]: remove dollar sign as it's not needed and should not be copied either.

Comment (simonschaufi) [inline README.md:13]: remove dollar sign as it's not needed and should not be copied either.

**Review by simonschaufi — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (simonschaufi) [inline README.md]:
```suggestion
composer require punktde/neos-environment-display
```

**Review by simonschaufi — CHANGES_REQUESTED** [review]

**Review by simonschaufi — CHANGES_REQUESTED** [review]
the dollar sign is not needed and should not be copied either

**Comment by crydotsnake** [comment]
> the dollar sign is not needed and should not be copied either

Thank you ! :)

**Pull request closed without merging**